BookingProject/BookingWebAPI/BookingWebApi.Domain/Entities/Apartament.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Domain.Entities
+{
+    public class Apartament
+    {
+        public int ApartamentId { get; set; }  
+        public string Address { get; set; }
+        public double Area { get; set; }
+        public double Latitude { get; set; }
+        public double Longitude { get; set; }       
```

Comment:
when you don't need fast calculation better use decimal

Reply:
Fixed